src/charm.py

```diff
@@ -626,7 +793,7 @@ def _restart_traefik(self):
                 _TRAEFIK_SERVICE_NAME: {
                     "override": "replace",
                     "summary": "Traefik",
-                    "command": "/usr/bin/traefik",
+                    "command": self._bin_path,
```

Comment:
Is there a reason to change this? Sure, `_bin_path` is also re-used in `.version`, but given that it's only in a couple of places, leaving it explicit is 50/50.

Reply:
The counterpoint to this is that variables which are used in very few places may require the author to mentally "bounce" around in the codebase to keep track of where they are, and it's a hardcoded literal there also.

Considering that "upstream" images may use, say, `/bin/foo` and ROCKs are almost certain to use `/usr/bin/foo`, if we're going to have something in the code (as a property or method, rather than attribute), it may as well wait for the container to be up and actually check the filesystem for where the path is.

It's more of a nit than anything, but a class-level private attribute for a hardcoded string may as well be a constant like other container paths (`_STATIC_CONFIG_DIR`, for example, though I have no idea why _that_ is private either), leave it hardcoded, or actually look in the container for it.

Reply:
Moved to module level attr for consistency.
Let's leave the `_` and the remaining class attributes for another time.